docs/pages/store/reference/misc.mdx

```diff
@@ -269,6 +269,62 @@ function slice32(bytes memory data, uint256 start) internal pure returns (bytes3
 | -------- | --------- | -------------------------------------------------------------------------- |
 | `output` | `bytes32` | The extracted bytes32 value from the specified position in the bytes blob. |
 
+## FieldLayout_Empty
```

Comment:
I have no idea why these are showing up here. I've tried clearing foundry output/cache and rebuilding and regenerating docs, but no luck.